pkgs/development/python-modules/accelerate/default.nix

```diff
@@ -89,6 +89,10 @@ buildPythonPackage rec {
   ] ++ lib.optionals (stdenv.isDarwin && stdenv.isx86_64) [
     # RuntimeError: torch_shm_manager: execl failed: Permission denied
     "CheckpointTest"
+  ] ++ lib.optionals stdenv.isDarwin [
+    # CppCompileError: C++ compile error
+    # OpenMP support not found.
+    "test_convert_to_fp32"
```

Comment:
How about adding `llvmPackages.openmp` to `buildInputs`?

Reply:
Indeed it works ! Running `nixpkgs-review` right now.